server/ntb/commands/fix_events_moment_timezone_2023.py

```diff
@@ -0,0 +1,57 @@
+import superdesk
+
+from datetime import datetime, timedelta
+from flask import current_app as app
+
+
+class FixEventsCommand(superdesk.Command):
+    """Fix events affected by moment timezone issue
+
+    which were saved during summer time using the offset of winter time (or vice versa in Australia).
+    """
+
+    option_list = [
+        superdesk.Option("--start", "-s", default="2023-03-26T01:00:00+00:00"),
+        superdesk.Option("--end", "-e", default="2023-10-29T01:00:00+00:00"),
+        superdesk.Option("--updated", "-u", default="2023-03-20T17:00:00+00:00"),
+        superdesk.Option("--offset", "-o", default="-1"),
+    ]
+
+    def run(self, start, end, updated, offset):
+        start = datetime.fromisoformat(start)
+        end = datetime.fromisoformat(end)
+        updated = datetime.fromisoformat(updated)
+        offset = timedelta(hours=int(offset))
+
+        app.logger.info(
+            "Updating events from %s to %s not updated after %s", start, end, updated
+        )
+
+        query = {
```

Comment:
Any particular reason to use Mongo and not Elastic? `dates.start` and `dates.end` are indexed, but `_updated` and `ingest_provider` isn't.

Reply:
not really, it's affecting 2k events for ntb so shouldn't be a big deal